boards/nrf52840dk/include/arduino_iomap.h

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (C) 2023 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf52840dk
+ * @{
+ *
+ * @file
+ * @brief       Mapping from MCU pins to Arduino pins
+ *
+ * You can use the defines in this file for simplified interaction with the
+ * Arduino specific pin numbers.
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
```

Comment:
You could put your name here :)

Reply:
Makes sense :) While this started as copy paste, the I/O mapping is indeed a bit different :/